src/utils.ts

```diff
@@ -15,3 +22,24 @@ export const getLeetcodeProfileURL = (username: string): string => {
 export const getGithubProfileURL = (username: string): string => {
   return `https://github.com/${username}`;
 };
+
+export const assertTypeAndOrderingIntegrity = (
```

Comment:
if this fails what happens? and is there ever a reason it could? or is this only for development/debugging?

Reply:
Yeah this is only for development